tests/package.json

```diff
@@ -12,7 +12,7 @@
   "scripts": {
     "test": "mocha --reporter mochawesome",
     "test-bail": "mocha -b --reporter mochawesome",
-    "test-blender28": "mocha -b --reporter mochawesome -g blender28",
+    "test-blender": "mocha -b --reporter mochawesome -g blender",
```

Comment:
I think this line is a holdover of the choice between 2.79 vs 2.80 vs both.  It's effectively the same as `test-bail` now, and can probably be removed.

Reply:
I can remove it. The only difference is it doesn't run the ["General" tests](https://github.com/KhronosGroup/glTF-Blender-IO/blob/6aa052af7d711267a83779b71cef0254bcb574e2/tests/test/test.js#L207-L216).

Reply:
Removed.